frontend/website-redesign/src/pages/Index.re

```diff
@@ -58,6 +58,23 @@ let make = () => {
           },
         |]
       />
+      <FeaturedSingleRow
+        row={
+          FeaturedSingleRow.Row.rowType: ImageLeftCopyRight,
```

Comment:
nice! so we can just specify which one we want in the future and create new ones

Reply:
Super clean.